tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -81,25 +84,62 @@ spec:
               - name: scheduler
                 args:
                   - --kube-api-qps=$(params.scheduler-qps)
+        etcd:
+          spec:
+            nodeSelector: 
+              node.kubernetes.io/instance-type: $(params.cp-instance-type)
+            containers: 
+            - name: etcd
+              args:
+                - --quota-backend-bytes=8589934592
       EOL
+      aws sts get-caller-identity
+      #Use service account attached to pod
+      # TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+      # kubectl config set-credentials tekton --token=$TOKEN
+      # kubectl config set-context --current --user=tekton
```

Comment:
are these intentionally commented out?

Reply:
good catch. that's was from my debugging. I will strip them out.